parts/k8s/kubernetesjumpboxcustomdata.yml

```diff
@@ -0,0 +1,15 @@
+#cloud-config
+
+write_files:
+
+- path: "/home/{{WrapAsVariable "jumpboxUsername"}}/.kube/config"
+  permissions: "0744"
+  encoding: "gzip"
+  owner: "root"
+  content: |
+    {{GetKubeConfig}}
+
+runcmd:

```

Comment:
Add this beauty:

`retrycmd_if_failure() { retries=$1; wait=$2; shift && shift; for i in $(seq 1 $retries); do ${@}; [ $? -eq 0  ] && break || sleep $wait; done; echo Executed \"$@\" $i times; }`

And then change the curl statement accordingly, e.g.:

`retrycmd_if_failure 10 5 curl -LO https://storage.googleapis.com/kubernetes-release/release/v{{.OrchestratorProfile.OrchestratorVersion}}/bin/linux/amd64/kubectl`